src/platform_impl/linux/x11/mod.rs

```diff
@@ -185,15 +187,16 @@ impl<T: 'static> EventLoop<T> {
 
         let poll = Poll::new().unwrap();
         let waker = Arc::new(Waker::new(poll.registry(), USER_REDRAW_TOKEN).unwrap());
-        let queue = Arc::new(NotificationQueue::new(waker));
 
         poll.registry()
             .register(&mut SourceFd(&xconn.x11_fd), X_TOKEN, Interest::READABLE)
             .unwrap();
 
-        let (user_sender, user_channel) = channel(queue.clone(), NotificationId::gen_next());
+        //let (user_sender, user_channel) = channel(queue.clone(), NotificationId::gen_next());
+        let (user_sender, user_channel) = std::sync::mpsc::channel();
 
-        let (redraw_sender, redraw_channel) = channel(queue, NotificationId::gen_next());
+        //let (redraw_sender, redraw_channel) = channel(queue, NotificationId::gen_next());
+        let (redraw_sender, redraw_channel) = std::sync::mpsc::channel();
```

Comment:
Don't need these comments here, the old code is preserved by Git 😉

Reply:
Whoops, that one slipped past me. Thanks for pointing it out!